lib/content-services/src/lib/content-metadata/components/content-metadata/content-metadata.component.html

```diff
@@ -20,113 +53,183 @@
-                    [classifiableChanged]="classifiableChanged"
-                    (categoriesChange)="storeCategoriesToAssign($event)">
-                </adf-categories-management>
-            </div>
+                <div *ngIf="!categories.length && !editableCategories" class="adf-metadata-no-catagories-added">
+                    {{ 'CATEGORIES_MANAGEMENT.NO_CATEGORIES_ADDED' | translate }}
+                </div>
+                <div *ngIf="editableCategories" class="adf-metadata-categories-header">
+                    <adf-categories-management
+                        [(categoryNameControlVisible)]="categoryControlVisible"
+                        [disableRemoval]="saving"
+                        [categories]="categories"
+                        [managementMode]="categoriesManagementMode"
+                        [classifiableChanged]="classifiableChanged"
+                        (categoriesChange)="storeCategoriesToAssign($event)">
+                    </adf-categories-management>
+                </div>
+            </mat-expansion-panel>
         </ng-container>
-        <ng-container *ngIf="expanded">
             <ng-container *ngIf="groupedProperties$ | async; else loading; let groupedProperties">
                 <div *ngFor="let group of groupedProperties; let first = first;"
                      class="adf-metadata-grouped-properties-container">
-                    <mat-expansion-panel *ngIf="showGroup(group) || editable"
-                                         [attr.data-automation-id]="'adf-metadata-group-' + group.title"
-                                         [expanded]="canExpandTheCard(group) || !displayDefaultProperties && first">
+                    <mat-expansion-panel
+                                    [attr.data-automation-id]="'adf-metadata-group-' + group.title"
+                                    [expanded]="canExpandTheCard(group) || !displayDefaultProperties && first || group.expanded"
+                                    (opened)="group.expanded = true"
+                                    (closed)="group.expanded = false"
+                                    hideToggle>
                         <mat-expansion-panel-header>
-                            <mat-panel-title>
-                                {{ group.title | translate }}
-                            </mat-panel-title>
+                            <div class="adf-metadata-properties-panel-content">
+                                <mat-icon>
+                                    {{ group.expanded ? 'expand_more' : 'chevron_right'}}
+                                </mat-icon>
+                                <mat-panel-title class="adf-metadata-properties-title">
+                                    {{ group.title | translate }}
+                                </mat-panel-title>
+                            </div>
+                            <button *ngIf="!group.editable && !readOnly && hasAllowableOperations()"
```

Comment:
please try to use variable instead of calling hasAllowableOperations function because we should to avoid calling functions in html

Reply:
@Yasa-Nataliya same here, you liked it but I can't see changes